controllers/userAuthControllers.js

```diff
@@ -0,0 +1,67 @@
+import { User } from '../models/sequelize.js';
+import bcrypt from 'bcrypt';
+import { jwtTokens } from '../utils/jwt-helpers.js'
+import jwt from 'jsonwebtoken';
+
+const registerUser = async (req, res) => {
+    const { username, password, email, role } = req.body;
```

Comment:
Move all the logic to the services folder. Controllers should not have this much logic.